client/components/Account/AccountPage.vue

```diff
@@ -31,13 +56,56 @@ import LogoutForm from '@/components/Account/LogoutForm.vue';
 export default {
   name: 'AccountPage',
   components: {
+    ChangeAccountNameForm,
+    AddCredentialForm,
     ChangeUsernameForm,
     ChangePasswordForm,
     DeleteAccountForm,
-    LogoutForm
+    LogoutForm,
```

Comment:
[nit] no need for ',' at the end

Reply:
This is an example of a [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas). I also used to hate them for being "unnecessary" but more recently I have come to appreciate them a lot.

The benefit is that if you want to reorder elements or add one to the end, you can do so without changing the existing line. It's especially convient with version-controlled projects because the diffs only show the lines that actually had a meaningful change rather than adding/removing trailing commas.

As you can see I'm still somewhat inconsistent in implementing them, but I do try to generally use them as much as possible.